tests/ui/lifetimes/elided-lifetime-in-path-details.tied.stderr

```diff
@@ -0,0 +1,179 @@
+error: hidden lifetime parameters in types are deprecated
+  --> $DIR/elided-lifetime-in-path-details.rs:13:5
+   |
+LL |     ContainsLifetime
+   |     ^^^^^^^^^^^^^^^^
```

Comment:
Surprised that these lints don't have the "expected lifetime parameter" primary span label.

Reply:
Ah. Good call. With the above suggestion, it now looks like

```
error: hidden lifetime parameters in types are deprecated
 --> /tmp/smol.rs:3:35
  |
3 | fn top_level_to_nested(v: &u8) -> ContainsLifetime {
  |                           -       ^^^^^^^^^^^^^^^^ expected lifetime parameter
  |                           |
  |                           the lifetime comes from here
  |
help: indicate the anonymous lifetime
  |
3 | fn top_level_to_nested(v: &u8) -> ContainsLifetime<'_> {
  |                                                   ++++
```